cxx-qt-gen/src/syntax/attribute.rs

```diff
@@ -0,0 +1,144 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::syntax::path::path_compare_str;
+use std::iter::FromIterator;
+use syn::{
+    ext::IdentExt,
+    parenthesized,
+    parse::{Parse, ParseStream},
+    punctuated::Punctuated,
+    spanned::Spanned,
+    token::Comma,
+    token::Paren,
+    Attribute, Error, Ident, Result,
+};
+
+/// Representation of a list of idents in an attribute, eg attribute(A, B, C)
+pub struct AttributeList {
+    pub bracket: Paren,
```

Comment:
brackets `[` and parens `(` are not the same things

Reply:
Changed to paren